cypress/integration/04 - server.ts

```diff
@@ -0,0 +1,63 @@
+describe("Server", () => {
+    const { serverUrl } = Cypress.env();
+
+    it("can properly respond to valid and invalid requests", () => {
+        cy.request({ url: `${serverUrl}/`, method: "GET", failOnStatusCode: false }).then(
+            (response) => expect(response.status).to.eq(405) // method not allowed
+        );
+
+        cy.request({ url: `${serverUrl}/`, method: "PUT", failOnStatusCode: false }).then(
+            (response) => expect(response.status).to.eq(405) // method not allowed
+        );
+
+        cy.request({ url: `${serverUrl}/`, method: "OPTIONS" });
+
+        cy.request({ url: `${serverUrl}/`, method: "POST", failOnStatusCode: false }).then(
+            (response) => expect(response.status).to.eq(400) // bad request (no json)
+        );
+
+        cy.request({
```

Comment:
This is just a simple test where "valid JSON" is sent, but not for an actual method.